localstack/services/lambda_/invocation/docker_runtime_executor.py

```diff
@@ -84,7 +87,7 @@ def docker_platform(lambda_architecture: Architecture) -> DockerPlatform | None:
 
 
 def get_image_name_for_function(function_version: FunctionVersion) -> str:
-    return f"localstack/lambda-{function_version.id.qualified_arn().replace(':', '_').replace('$', '_').lower()}"
+    return f"localstack/prebuild-lambda-{function_version.id.qualified_arn().replace(':', '_').replace('$', '_').lower()}"
```

Comment:
sidenote: We might want to exclude these from `localstack update docker-images` in the future because it obviously won't be able to pull these local prebuilds:

```log
ERROR: '['docker', 'pull',
'localstack/prebuild-lambda-arn_aws_lambda_us-east-1_000000000000_function_test_lambda_x86_cdf6eb01__latest:la
test']': exit code 1; output: b"Error response from daemon: pull access denied for
localstack/prebuild-lambda-arn_aws_lambda_us-east-1_000000000000_function_test_lambda_x86_cdf6eb01__latest,
repository does not exist or may require 'docker login': denied: requested access to the resource is denied\n"
✖ Image
localstack/prebuild-lambda-arn_aws_lambda_us-east-1_000000000000_function_test_lambda_x86_cdf6eb01__latest:lat
est pull failed: Docker image
localstack/prebuild-lambda-arn_aws_lambda_us-east-1_000000000000_function_test_lambda_x86_cdf6eb01__latest:lat
est not found
```